### PR TITLE
Angular 8 Upgrade

### PR DESCRIPTION
**Pull request by 89snake89**

### CODE REVIEW

File: src/package.json [hunk 1]
```diff
@@ -9,20 +9,20 @@
   },
   "devDependencies": {
     "@angular/common": "^7.0.3",
```
Comment (rpaschoal) [inline src/package.json]: Shouldn't all these have been migrated too?

Comment (89snake89) [reply src/package.json]: Yes
I was figuring out a problem about Angular compiler/Typescript compatibility

Comment (89snake89) [reply src/package.json]: Migrated. 
There are still some issues with tests.

**Comment by rpaschoal** [comment]
I'll merge this into another branch so I can work on fixing the test failures and potentially migrating to Angular 9. Thank you for the headstart @89snake89 , hopefully soon we'll be able to merge all these into master 🎉

**Pull request merged**